src/pages/support/support.module.scss

```diff
@@ -0,0 +1,28 @@
+@import '~@bit/bit.base-ui.theme.colors';
+@import '~@bit/bit.base-ui.layout.breakpoints';
+@import '~@bit/bit.base-ui.constants.storage/_storage.module.scss';
+
+
+.supportCirclesBg {
+	background: url($staticStorageUrl + '/support-page/hero-background/support-hero-strips.svg'), url($staticStorageUrl + '/support-page/hero-background/support-hero-bg.svg');
+	background-repeat: no-repeat;
+	background-position: center bottom;
+	margin-bottom: 150px;
+	@media screen and (min-width: $br-md) {
+		margin-bottom: 100px;
+	}
+}
+
+.supportPageStyles {
+	h1, h2, h3 {
+		color: var(--text-highlight, inherit);
+	}
+}
+
+
+.foldMargin {
+	transform: translateY(90px);
```

Comment:
odd choice

Reply:
this is so that the image is a bit out of the curved line. do you have a better solution?